.gitignore

```diff
@@ -0,0 +1,5 @@
+diwako_cbrn.vr/.editorconfig
+diwako_cbrn.vr/initServer.sqf
+diwako_cbrn.vr/init.sqf
+diwako_cbrn.vr/description.ext
```

Comment:
This is more of a discussion point i think.

Should mission.sqm be added as well seeing description.ext is in there as well.

Reply:
The community I'm has a special framework that we run in all our missions and does a bunch of really useful stuff for admins as well. I added this to my fork to make it shut up about the changes in there.

I'm gonna commit a delete order for this file in a commit. It's not supposed to be here. :)

Reply:
Ah, yeah i can see that being a thing. It is always a bit annoying working on standalone mission scripts, but the only real way to test them out is making an actual mission. so you have to mix and match and it becomes annoying fast.